src/main/java/com/google/devtools/build/lib/rules/android/AarImport.java

```diff
@@ -65,6 +69,11 @@ public class AarImport implements RuleConfiguredTargetFactory {
   private static final String MERGED_JAR = "classes_and_libs_merged.jar";
   private static final String PROGUARD_SPEC = "proguard.txt";
 
+  private static final ParamFileInfo WORKERS_FORCED_PARAM_FILE_INFO =
+      ParamFileInfo.builder(ParameterFile.ParameterFileType.UNQUOTED)
+          .setUseAlways(true)
```

Comment:
Why is this unconditionally set to true?

Reply:
same as BusyBoxActionBuilder.